go.mod

```diff
@@ -63,6 +63,7 @@ require (
 	github.com/pkg/errors v0.9.1
 	github.com/polyfloyd/go-errorlint v0.0.0-20210722154253-910bb7978349
 	github.com/prometheus/procfs v0.6.0 // indirect
+	github.com/quasilyte/go-ruleguard v0.3.13 // indirect
```

Comment:
Without this line, the `ruleguard` functional test fails. Specifically in [test/testdata/configs/ruleguard/dup.go](https://github.com/golangci/golangci-lint/pull/2308/files#diff-e4e65433aef638f729eaf0ac7108a187f389ad64d64ae7377792b50a5f735fd2R3), ruleguard is not able to import `github.com/quasilyte/go-ruleguard/dsl`

```
go get github.com/quasilyte/go-ruleguard
```

Reply:
this happens because you put the `ruleguard` package to `testdata` folder, go ignores it. Please move under `test` folder

Reply:
Ah thank you, that makes sense now. I am updating the PR.